packages/back-end/src/jobs/proxyUpdate.ts

```diff
@@ -146,13 +141,52 @@ export async function queueProxyUpdate(
       continue;
     }
 
-    await queueSingleProxyUpdate(connection);
+    if (IS_CLOUD) {
+      // Always fire webhook to GB Cloud Proxy for cloud users
+      await queueSingleProxyUpdate(connection, true);
+    }
+    // If connection (cloud or self-hosted) specifies an (additional) proxy host, fire webhook
+    await queueSingleProxyUpdate(connection, false);
   }
 }
 
-function connectionSupportsProxyUpdate(connection: SDKConnectionInterface) {
-  // note: sseEnabled indicates that we are using Cloud Proxy behind the scenes
-  if (IS_CLOUD) return !!connection.sseEnabled;
-
+function connectionSupportsProxyUpdate(
+  connection: SDKConnectionInterface,
+  useCloudProxy: boolean = false
+) {
+  if (useCloudProxy) {
+    return IS_CLOUD && !!connection.sseEnabled;
```

Comment:
Do we still want that `sseEnabled` check here?

Reply:
we will probably remove this once SSE rollout goes live